src/components/DryRun/DryRun.tsx

```diff
@@ -33,15 +31,16 @@ interface DryRunProps {
   query: string;
   showDialog: boolean;
   setShowDialog: Dispatch<SetStateAction<boolean>>;
-  asyncEvents: WebhookEventTypeAsyncEnum[];
   setResult: Dispatch<SetStateAction<string>>;
+  data: WebhookFormData;
 }
 
-export const DryRun: React.FC<DryRunProps> = ({
+const DryRun: React.FC<DryRunProps> = ({
   setResult,
   showDialog,
   setShowDialog,
   query,
+  data: { syncEvents },
```

Comment:
question: why do you pass hole data even if that component uses only syncEvents ?

Reply:
changed